apps/src/code-studio/pd/workshop_dashboard/attendance/session_attendance_row.jsx

```diff
@@ -25,13 +25,18 @@ const SessionAttendanceRow = React.createClass({
       enrollment_id: React.PropTypes.number.isRequired,
       user_id: React.PropTypes.number,
       in_section: React.PropTypes.bool.isRequired,
-      attended: React.PropTypes.bool.isRequired
+      attended: React.PropTypes.bool.isRequired,
+      puzzles_completed: React.PropTypes.number.isRequired
     }).isRequired,
```

Comment:
as an aside, it makes me a little sad to see underscored properties here in JS-land; it might be worth (probably in a separate PR) updating these to camelCase